grid/recruitment/list-jobs/maps/greenhouse.suma

```diff
@@ -0,0 +1,60 @@
+profile = "recruitment/list-jobs@1.0"
+provider = "greenhouse"
+
+map ListJobs {
+  // Greenhouse to Superface
+  JOB_STATES_G2S = {
+    "open": "published",
+    "closed": "closed",
+    "draft": "draft"
+  }
+
+  // Superface to Greenhouse
+  JOB_STATES_S2G = {
+    "published": "open",
+    "closed": "closed",
+    "draft": "draft"
+  }
+
+  page = input.page || 1
+  limit = input.limit || 100
+  
+  http GET "/v1/jobs" {
+    security "apiKeyBasicAuth"
+  
+    request {
+      query {
+        page = page
+        per_page = limit
+        status = JOB_STATES_S2G[input.state]
```

Comment:
issue: Since state is not mandatory, I may end up passing `JOB_STATES_S2G[undefined]` here.

Reply:
In which case it evaluates to `undefined` and isn't passed in the `query` at all, right?

Reply:
Right, it should, I just don't like that it relies on implicit behavior. Feel free to ignore this suggestion then.

```suggestion
        status = input.state && JOB_STATES_S2G[input.state]
```

(even better would be to assign state above the http call)